app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/CreateNoteUpload.java

```diff
@@ -187,10 +187,11 @@ private static String getCreateNoteText(CreateNote note)
 			}
 			else
 			{
-				return "for " + getAssociatedElementString(note) + " :\n\n" + note.text;
+				return "for " + getAssociatedElementString(note) +
+						" via "+ ApplicationConstants.USER_AGENT+":\n\n" + note.text;
 			}
 		}
-		return note.text;
+		return "via "+ ApplicationConstants.USER_AGENT+":\n\n" + note.text;
```

Comment:
Why is this relevant?

Reply:
I just thought it might be useful if somebody wants to review notes created by StreetComplete with https://ent8r.github.io/NotesReview/
But it is totally fine if you think that this should not be added